libraries/SITL/SIM_AirSim.cpp

```diff
@@ -0,0 +1,158 @@
+/* 
+	Simulator Connector for AirSim
+*/
+
+#include "SIM_AirSim.h"
+
+#include <stdio.h>
+#include <arpa/inet.h>
+#include <errno.h>
+
+#include <AP_HAL/AP_HAL.h>
+
+extern const AP_HAL::HAL& hal;
+
+using namespace SITL;
+
+AirSim::AirSim(const char *home_str, const char *frame_str) :
+	Aircraft(home_str, frame_str),
+	sensor_sock(true),
+	control_sock(true),
+	last_timestamp(0),
```

Comment:
No need to zero these; we will never make one of these on the stack and elsewhere it gets zeroed.